### PR TITLE
olsrd: fix build

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/olsrd.rb [hunk 1]
```diff
@@ -22,13 +22,45 @@ class Olsrd < Formula
 
   depends_on "coreutils" => :build # needs GNU cp
```
Comment (cho-m) [inline Formula/olsrd.rb:23]: Can we assume GNU `cp` exists on all Linux?

Comment (danielnachun) [reply Formula/olsrd.rb]: Yes, I think we can.  I've changed this dependency to be macOS-only.

File: Formula/olsrd.rb [hunk 2]
```diff
@@ -62,6 +96,6 @@ def startup_plist
   end
 
   test do
-    assert_match version.to_s, pipe_output("#{sbin}/olsrd")
+    assert_match version.to_s, pipe_output("#{bin}/olsrd")
```
Comment (Bo98) [inline Formula/olsrd.rb:99]: Is there a reason for changing this. I don't think we have a policy to avoid `sbin`.

Comment (danielnachun) [reply Formula/olsrd.rb:99]: Let me see if I can change `SBINDIR` to point to `sbin` instead of `bin`.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**